test/Identity.test.js

```diff
@@ -37,36 +37,45 @@ contract("Identity", function (accounts) {
             const data = this.testProxy.contract.methods.callMe().encodeABI();
             await shouldRevert(
                 this.identity.execute(this.testProxy.address, 0, data),
-                "Requester must have an ACTION purpose"
+                "Sender must have ACTION purpose"
             );
         })
 
 
         it('should execute contract method for identity ACTION key', async function () {
             const data = this.testProxy.contract.methods.callMe().encodeABI();
             await this.identity.addKey(addressToBytes32(accounts[1]), ACTION, 1);
-            shouldSucceed(await this.identity.execute(this.testProxy.address, 0, data, {from: accounts[1]}));
+            await shouldSucceed(this.identity.execute(this.testProxy.address, 0, data, {from: accounts[1]}));
             const numOfCalls = await this.testProxy.getCallsFrom(this.identity.address);
             assert.equal(1, numOfCalls.toNumber());
         })
 
 
+        it('should transferEth using the a identity ACTION key', async function () {
+            await this.identity.addKey(addressToBytes32(accounts[1]), ACTION, 1);
+            await shouldSucceed(this.identity.transferEth(accounts[2], 1, "0x1234", {from: accounts[1]}));
```

Comment:
Do you check if `account[2]` has one more ETH on his account after the method call?